docs/reference/commandline/build.md

```diff
@@ -373,6 +374,21 @@ the command line.
 > repeatable builds on remote Docker hosts. This is also the reason why
 > `ADD ../file` will not work.
 
+### Build with Secrets
```

Comment:
would be nice to have rhel subs while building daemon wide (but we also need them at run time :( and we currently inject those)